app/views/idv/doc_auth/document_capture.html.erb

```diff
@@ -79,3 +101,4 @@
 <%= render 'idv/doc_auth/start_over_or_cancel' %>
 <%= javascript_pack_tag 'image-preview' %>
 </div>
+<%= javascript_pack_tag 'document-capture' %>
```

Comment:
I think this may have been the artifact of a rebase after #3922 was merged. I don't think this change should be here. In #3922, it [was moved higher in the file](https://github.com/18F/identity-idp/pull/3922/files#diff-1cefa3c952d759083c2936beb679b2a3), and now there's two script tags that unfortunately conflict with one another 😬

Reply:
Sneaking in a fix at https://github.com/18F/identity-idp/pull/3964#issuecomment-663207142